Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:testing
+
+MAINTAINER Raniere Silva
+
+RUN apt-get -y update && apt-get -y install curl r-base r-base-dev
+RUN curl https://repo.continuum.io/archive/Anaconda3-4.2.0-Linux-x86_64.sh -o Anaconda3-4.2.0-Linux-x86_64.sh && bash Anaconda3-4.2.0-Linux-x86_64.sh -b
```

Comment:
@wking Thanks very much for the suggestion. I'm going to wait until we have a consensus at http://lists.software-carpentry.org/pipermail/maintainers/2016-November/000346.html before back to work on this.